tests/rkt_tests.go

```diff
@@ -124,11 +124,19 @@ func spawnOrFail(t *testing.T, cmd string) *gexpect.ExpectSubprocess {
 	return child
 }
 
+// waitOrFail waits for the child to exit, draining all its output.
+// If a non-negative return value is provided, child exit status must match.
 func waitOrFail(t *testing.T, child *gexpect.ExpectSubprocess, expectedStatus int) {
+	bufOut := []string{}
+	ttyIn, ttyOut := child.AsyncInteractChannels()
+	close(ttyIn)
+	for line := range ttyOut {
```

Comment:
(I guess) this relies on the fact that the `ttyOut` chan will be closed when a real err happens or `io.EOF` in https://github.com/ThomasRooney/gexpect/blob/8e96656e8fb794d716ee75b5a8cecbc15cde4393/gexpect.go#L167. I think we should document this here and in gexpect.